tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -131,14 +130,32 @@ df_exp[!, :cell] = 1:n_cells
 )
 ```
 
-By design, we mannually divide the 60 cells into two groups. the first 3 geme features of the first 30 cells have mean 10, while those of the last 30 cells have mean 10.
+By design, we mannually divide the 60 cells into two groups. the first 3 gene features of the first 30 cells have mean 10, while those of the last 30 cells have mean 10.
 These two groups of cells differ in the expression of genes.
 
-To make these differences, we draw sample values from the $3 \times 3$ multivariate normal distribution (MND) with mean 10 to fill in the first and last 3 geme features;
+To make these differences, we draw sample values from the $3 \times 3$ multivariate normal distribution (MND) with mean 10 to fill in the first and last 3 gene features;
 for the remaining features, we draw their values from zero-mean MND.
-We then use `hcat` and `vcat` to concatenate the columns and rows to make a $9 \times 60$ matrix, and then transpose it to form a $60 \times 9$ feature matrix, with rows the instance (cells) and columns the features (gemes).
-Finally we turn this matrix into a `DataFrame` type, and add an extra ID column `cell` to number the cells.
-To visualize the $60 \times 10$ DataFrame, we use our custom-built function `vlplot()`.
+We then use `hcat` and `vcat` to concatenate the columns and rows to make a $9 \times 60$ matrix, and then transpose it to form a $60 \times 9$ feature matrix, with rows the instance (cells) and columns the features (genes).
+Finally we turn this matrix into a `DataFrame` type. The data matrix `X` now has dimension $m=60 \times D=9$ where `N=60` is the number of cells (instances), and `D=9` is the number of genes (features). This shape aligns with the common use of `design matrix` in data science. However, in order to align with our p-PCA formula, we need to transpose this dataframe into `dimension x number of instance`:
+
+```Julia
+mat_exp = transpose(Matrix(df_exp))
+```
+
+To visualize the $D=9 \times N=60$ data matrix `mat_exp`, we use our custom-built function `vlplot()`. An alternative is to use the `heatmap` plot.
```

Comment:
Just use one thing (and IMO just remove VegaLite).

Reply:
yes, all appearances of `vplot` have been removed.